source/reference/verification.txt

```diff
@@ -0,0 +1,114 @@
+
+====================
+Verify Data Transfer
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Following a sync, verify the successful transfer of your data before you switch
+your application from using the source cluster to the destination cluster. You
+can verify your data transfer using document counts, hash comparison, document
+comparison, or the Migration Verifier.
+
+Use Cases
+---------
+
+You should verify your data after every sync. This is important in
+cases where you plan to move your application load from the source to the
+destination cluster.
+
+Verification methods include the comparison of
+:ref:`Document Counts <c2c-verify-doc-counts>`,
+comparison of :ref:`Hash Comparison <c2c-verify-hash-comp>`,
+scripting :ref:`Document Comparisons <c2c-verify-doc-comp>`,
+and the :ref:`Migration Verifier <c2c-verify-migration-verifier>`.
+The specific method you use to verify your data depends on your application
+workload and the complexity of the data.
+
+.. _c2c-verify-method:
+
+Tasks
```

Comment:
Thanks for this refactor. I think this is a big improvement to the presentation.